entrypoint.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 source /venv/bin/activate
-exec "$@"
\ No newline at end of file
+exec "$@"
+exec bash
```

Comment:
I think this line `exec bash` will not be executed if the user provides custom commands. May be we could add a conditional statement to execute /bin/bash only if no command is provided so that we can still support both options (allow the container to default to an interactive shell)